bndtools.jareditor/src/bndtools/jareditor/internal/TemporaryFile.java

```diff
@@ -63,7 +64,14 @@
 		}
 	}
 
-	private static Result<IFolder, String> select() throws CoreException, IOException {
+	private static Result<IFolder, String> selectTempProject() throws CoreException, IOException {
+		IProject project = tempProject.getJavaProject()
+			.getProject();
+
+		return Result.ok(project.getFolder("temp"));
+	}
+
+	private static Result<IFolder, String> selectProject() throws CoreException, IOException {
```

Comment:
I don't see where this private method `selectProject` is ever called.

Reply:
It can be removed, I just left it in case we needed both mechanisms for some reason.